### PR TITLE
/cacert endpoint

### LINKED ISSUE #133: api: add /cacert endpoint serving MITM CA

### PR DESCRIPTION
**Pull request by mmatczuk**
Fixes #133

### CODE REVIEW

File: e2e/forwarder/service.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ const (
 	HttpbinServiceName       = "httpbin"
 )
 
+var enabled = "true"
```
Comment (Choraden) [inline e2e/forwarder/service.go:26]: How about `const`?

File: e2e/tests/flag_test.go [hunk 2]
```diff
@@ -97,6 +98,24 @@ func TestFlagMITM(t *testing.T) {
 		ExpectHeader("test-resp-add", "test-resp-value")
 }
 
+func TestFlagMITMGenCA(t *testing.T) {
+	r := newClient(t, proxyAPI).GET("/cacert").ExpectStatus(http.StatusOK)
```
[the remark this answers is not in the document]
Comment (mmatczuk) [reply e2e/tests/flag_test.go:102]: ie to erase proxy function?

[the remark this answers is not in the document]
Comment (Choraden) [reply e2e/tests/flag_test.go:102]: yes, erase proxy function from transport.

File: e2e/forwarder/service.go (hunk 1 above)
Comment (mmatczuk) [reply e2e/forwarder/service.go:26]: Done

File: e2e/tests/flag_test.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (mmatczuk) [reply e2e/tests/flag_test.go:102]: Done

```
	r := newClient(t, proxyAPI, func(tr *http.Transport) {
		tr.Proxy = nil
	}).GET("/cacert").ExpectStatus(http.StatusOK)
```

**Pull request merged**